### PR TITLE
police comms to CAS and MORTAR

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
QoL improvement refactors the MP channel to Cas/mortar "fire support one"
adds the headset to the marine clothing vendor with the regular marine generic headset
and adds 3 of these to the mortar crate
channel was also added to command and the PO role

## Why It's Good For The Game
https://github.com/tgstation/TerraGov-Marine-Corps/projects/1#card-50871965

## Changelog
:cl:
tweak: MP headset to fire support specialist headset
remove: master of arms headset
fix: Cas and Mortar dedicated channel is now in place
/:cl:

### CODE REVIEW

**Comment by fernandos13** [comment]
not working YET dont merged it as is even it if compiles

**Comment by BraveMole** [comment]
If it's still a work in progress, you can add [WIP] to your PR name, that'll give it an automatic tag

**Comment by fernandos13** [comment]
thanks!

**Comment by fernandos13** [comment]
working now fixed

File: code/game/objects/items/radio/encryptionkey.dm [hunk 1]
```diff
@@ -12,12 +12,12 @@
 /obj/item/encryptionkey/engi
 	name = "Engineering Radio Encryption Key"
 	icon_state = "eng_cypherkey"
-	channels = list(RADIO_CHANNEL_ENGINEERING = TRUE)
+	channels = list(RADIO_CHANNEL_ENGINEERING = TRUE,RADIO_CHANNEL_CAS = TRUE)
 
-/obj/item/encryptionkey/sec
-	name = "Security Radio Encryption Key"
+/obj/item/encryptionkey/cas
+	name = "Fire support specialist Key"
```
Comment (SplinterGP) [inline code/game/objects/items/radio/encryptionkey.dm:18]:
```suggestion
	name = "Fire support radio encryption key"
```

File: code/game/objects/structures/mortar.dm [hunk 2]
```diff
@@ -442,3 +442,6 @@
 	new /obj/item/encryptionkey/engi(src)
 	new /obj/item/encryptionkey/engi(src)
 	new /obj/item/binoculars/tactical/range(src)
+	/obj/item/radio/headset/mainship/marine/generic/mmpo
+	/obj/item/radio/headset/mainship/marine/generic/mmpo
+	/obj/item/radio/headset/mainship/marine/generic/mmpo
```
Comment (SplinterGP) [inline code/game/objects/structures/mortar.dm:447]:
```suggestion
	new /obj/item/radio/headset/mainship/marine/generic/mmpo(src)
	new /obj/item/radio/headset/mainship/marine/generic/mmpo(src)
	new /obj/item/radio/headset/mainship/marine/generic/mmpo(src)
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]
all good to me

File: code/__DEFINES/radio.dm [hunk 3]
```diff
@@ -21,9 +21,9 @@
 #define RADIO_KEY_COMMAND "v"
 #define RADIO_TOKEN_COMMAND ":v"
 
-#define RADIO_CHANNEL_POLICE "Police"
-#define RADIO_KEY_POLICE "p"
-#define RADIO_TOKEN_POLICE ":p"
+#define RADIO_CHANNEL_CAS "fire support"
```
Comment (SplinterGP) [inline code/__DEFINES/radio.dm:24]:
```suggestion
#define RADIO_CHANNEL_CAS "Fire support"
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/items/radio/headset.dm [hunk 4]
```diff
@@ -515,6 +499,11 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	name = "marine generic radio headset"
 	icon_state = "headset_marine_generic"
 
+/obj/item/radio/headset/mainship/marine/generic/mmpo
```
Comment (SplinterGP) [inline code/game/objects/items/radio/headset.dm:502]: forgot typepath

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by SplinterGP** [comment]
you just need to replace the old typepath in maps with the new one, if you are in VSC should be easy as just searching the old typepath for the new one

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**